### PR TITLE
Docs: Add OpenLIT to Agent Observability doc along with notebook

### PR DESCRIPTION
**Pull request by patcher9**
## Why are these changes needed?

Hi Team, I am one of the maintainers of [OpenLIT](https://github.com/openlit/openlit) and we support Obserability for Agents built using AG2, It is all OpenTelemetry-native so the traces and metrics can be sent to any platfrom like [Grafana](https://grafana.com/docs/grafana-cloud/monitor-applications/ai-observability/introduction/) or any [OSS OTel tools](https://opentelemetry.io/blog/2024/llm-observability/)

Figured the mention here might be useful for developers using AG2 hence adding the reference in Agent Observabiliy documentation page along with a notebook

## Checks

- [x] I've included any doc changes needed for https://ag2ai.github.io/ag2/. See https://ag2ai.github.io/ag2/docs/Contribute#documentation to build and test documentation locally.
- [x] I've made sure all auto checks have passed.

Previous PR - https://github.com/ag2ai/ag2/pull/56

### CODE REVIEW

**Review by mcam10 — APPROVED** [review]

**Comment by mcam10** [comment]
> @Hk669

Maybe this addition can be done in two PRs? One to add the notebook -- then one to update the docs?

**Comment by Hk669** [comment]
> > @Hk669
> 
> Maybe this addition can be done in two PRs? One to add the notebook -- then one to update the docs?

Single PR is ideal for us to easily track and maintain the changes. Thanks.

**Comment by Hk669** [comment]
> @mcam10 @Hk669 Continuing the discussion from previous [PR](https://github.com/ag2ai/ag2/pull/56) that I closed
> 
> I am able to add the notebook and I can see all the workflows are passing (In my forked repo, Check [here](https://github.com/patcher9/ag2/pull/6))
> 
> But when I update the file https://github.com/ag2ai/ag2/blob/main/website/docs/topics/llm-observability.md, The build fails as it says Im trying to modify a file. Can someone help me with updating that file?

I believe these docs are auto generated from the other files. im not 100% sure about it, @skzhang1 can you please help us on this. Thanks

**Comment by qingyun-wu** [comment]
@patcher9 could you let me know your discord id or email? Thank you!

File: notebook/agentchat_openlit.ipynb [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "front_matter": {
+      "description": "Use OpenLIT to easily monitor AI agents in production with OpenTelemetry.",
+      "tags": [
+        "integration",
+        "monitoring",
+        "debugging"
```
Comment (sonichi) [inline notebook/agentchat_openlit.ipynb:440]:
```suggestion
        "observability",
        "debugging"
```

**Comment by sonichi** [comment]
> @mcam10 @Hk669 Continuing the discussion from previous [PR](https://github.com/ag2ai/ag2/pull/56) that I closed
> 
> I am able to add the notebook and I can see all the workflows are passing (In my forked repo, Check [here](https://github.com/patcher9/ag2/pull/6))
> 
> But when I update the file https://github.com/ag2ai/ag2/blob/main/website/docs/topics/llm-observability.md, The build fails as it says Im trying to modify a file. Can someone help me with updating that file?

Could you show the build error? It may be simply a format issue.

**Review by skzhang1 — APPROVED** [review]

**Comment by qingyun-wu** [comment]
> > @patcher9 could you let me know your discord id or email? Thank you!
> 
> Hey @qingyun-wu, Not on discord but my email is [patcher@openlit.io](mailto:patcher@openlit.io)

Thank you @patcher9. All contributors need to sign a CLA. A CLA bot is not in place yet. Just sent you a CLA via DocuSign. Please sign it before the PR is merged! Thank you for your contribution!

**Review by qingyun-wu — APPROVED** [review]

File: notebook/agentchat_openlit.ipynb [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+      "metadata": {
+        "id": "pxawniRGYyj2"
+      },
+      "outputs": [],
+      "source": [
+        "import openlit\n",
+        "\n",
+        "from autogen import AssistantAgent, UserProxyAgent\n",
+        "\n",
+        "openlit.init()"
```
Comment (sonichi) [inline notebook/agentchat_openlit.ipynb:210]: Minor issue: The import and init statements appear twice in this cell and the next cell.

**Review by sonichi — APPROVED** [review]
LGTM.

**Review by skzhang1 — APPROVED** [review]

**Pull request merged**